packages/editor/src/Editor.stories.tsx

```diff
@@ -17,6 +17,13 @@ import { linkJson } from "./storyFixtures/link";
 import { pullQuoteJson } from "./storyFixtures/pullquote";
 import { imageJson } from "./storyFixtures/image";
 import { imageBreakoutJson } from "./storyFixtures/imagebreakout";
+import {
```

Comment:
Should import and add them through a loop somehow?

Reply:
There's only ever going to be 4